scripts/jenkins/cloud/ansible/roles/supportconfig/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+
+---
+
+- name: Install requirements for supportconfig
+  zypper:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ supportconfig_pkgs }}"
+
+- name: Generate supportconfig
+  shell: |
+    supportconfig -B {{ supportconfig_filename }}
+  args:
+    creates: "/var/log/nts_{{ supportconfig_filename }}.tbz"
+  failed_when: false
+
+- name: Collect supportconfig
```

Comment:
Couldn't you reuse the `jenkins_artifacts` role on the playbook? it does exactly the same thing plus ensures that `jenkins_artifacts_dir` exists.

Reply:
That's a good point. Let me try that.